modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-api -o yaml
+oc -n $NAMESPACE describe deploy virt-api
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+//VirtControllerDown Alert
+[id="VirtControllerDown_{context}"]
+== VirtControllerDown alert
+
+.Description
+
+If no detection of virt-controllers occurs in the past five minutes, then virt-controller deployment has a default replica of two pods.
+
+.Reason
+
+If the virt-controller fails, then VM lifecycle management tasks, such as launching a new VM instance or shutting down an existing VM instance, completely fail.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Check the status of the virt-controller deployment.
++
+[source,terminal]
+----
+oc get deployment -n $NAMESPACE virt-controller -o yaml
+----
+
+. Check the manager pod's logs to determine why creating the virt-controller pods fails.
++
+[source,terminal]
+----
+oc get logs <virt-controller-pod>
+----
+
+[NOTE]
+====
+Do not use a name for the virt-controller pod that is similar to virt-controller-7888c64d66-dzc9p.
+====
+
+There may be several pods that run virt-controller.
+
+.Resolution
+
+There are several reasons why detection of no running virt-controller occurs. Identify the root cause and take appropriate action.
+
+* Node resource exhaustion
+* Not enough memory on the cluster
+* Nodes are down
+* The API server overloads, such as when the scheduler is not 100% available)
+* Networking issues
+
+Otherwise, open a support issue and provide the information gathered in the troubleshooting process.
+
+//VirtControllerRESTErrorsBurst Alert
+[id="VirtControllerRESTErrorsBurst_{context}"]
+== VirtControllerRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls failed in virt-controller in the last five minutes.
+
+.Reason
+
+Virt-controller has potentially fully lost connectivity to the API server. This loss does not affect running workloads, but propagation of status updates and actions like migrations cannot occur.
+
+.Troubleshoot
+
+There are two common error types associated with virt-controller REST call failure:
+
+* The API server overloads, causing timeouts. Check the API server metrics and details like response times and overall calls.
+
+* The virt-controller pod cannot reach the API server. Common causes are:
+** DNS issues on the node
+** Networking connectivity issues
+
+.Resolution
+
+Check the virt-controller logs to determine if it cannot connect to the API server at all. If so, delete the pod to force a restart. The issue normally relates to DNS or CNI issues outside of the scope of this alert.
+
```

Comment:
We might want to document, what all information should be collected if a support case is deemed necessary.

Reply:
@dshchedr Would you be able to provide some insight as to what information would be good to collect for a support ticket?

In other places in these alerts, we've said "Open a support issue and provide the information gathered in the troubleshooting process."

Is that enough or should more specific information be listed?

Reply:
In general, checking logs might be enough. But I would also add verification of this possible reasons: 

* Node resource exhaustion
* Not enough memory on the cluster